WordPress/Classes/Models/Menu.m

```diff
@@ -16,4 +19,36 @@ + (NSString *)entityName
     return NSStringFromClass([self class]);
 }
 
++ (Menu *)newMenu:(NSManagedObjectContext *)managedObjectContext
+{
+    Menu *newMenu = [NSEntityDescription insertNewObjectForEntityForName:[Menu entityName] inManagedObjectContext:managedObjectContext];
+    return newMenu;
+}
+
++ (Menu *)defaultMenuForBlog:(Blog *)blog
+{
+    Menu *defaultMenu = nil;
+    for (Menu *menu in blog.menus) {
+        if (menu.menuID.integerValue == MenuDefaultID) {
+            defaultMenu = menu;
+            break;
+        }
+    }
+    return defaultMenu;
+}
+
++ (Menu *)newDefaultMenu:(NSManagedObjectContext *)managedObjectContext
+{
+    // Create a new default menu.
```

Comment:
Not really an issue but I think this comment can safely be removed given how clear the context is.